README.md

```diff
@@ -70,7 +72,7 @@ nix develop
 Run the following to format the entire repo and check your spelling before each PR:
 
 ```sh
-nix run .#pre-commit -L
+nix run .#pre-commit --print-build-logs
```

Comment:
Why change this?

Reply:
the full flag makes it self explanatory for nix noobies

Reply:
did we get any reports of this being confusing?

Reply:
my experience when starting and needing to lookup what `-L` stands for

Reply:
it's a though one because in practice you want to append `-L` after every single nix command, and people new to nix don't want to be typing a long flag after each command 🤔